installing_on_kubernetes/_topics/checking_readiness.md

```diff
@@ -0,0 +1,37 @@
+## Check readiness of the MIQ pods
+
+_**Note:**_ Please allow a few minutes for the application to start.  Initial install and upgrades take extra time due do database initialization.  HTTPS traffic will not be served until pods are running and ready.  The READY column denotes the number of replicas and their readiness state.  The operator deploys the following pods:
+
+        $ oc get pods
```

Comment:
Instead of indented blocks, can we use triple backticks?  This way we can use syntax highlighting such as in the section in the next doc that has yaml.

Reply:
We can switch, I used indented blocks because that was already here.

Reply:
Done